ReSharper.FSharp/src/FSharp.Psi.Features/src/QuickFixes/RemoveUnexpectedArgumentsFix.fs

```diff
@@ -0,0 +1,25 @@
+﻿namespace JetBrains.ReSharper.Plugins.FSharp.Psi.Features.Daemon.QuickFixes
+
+open JetBrains.ReSharper.Plugins.FSharp.Psi.Features.Daemon.Highlightings
+open JetBrains.ReSharper.Plugins.FSharp.Psi.Tree
+open JetBrains.ReSharper.Psi.Tree
+open JetBrains.ReSharper.Resources.Shell
+open JetBrains.ReSharper.Plugins.FSharp.Psi.Util
+
+type RemoveUnexpectedArgumentsFix(warning: NotAFunctionError) =
+    inherit FSharpQuickFixBase()
+        
+    let notAFunctionExpr = warning.NotAFunctionExpr
+    let unexpectedArgs = warning.UnexpectedArgs

```

Comment:
Maybe just `expr` and `args`? The rest seems clear from the context and code below would be easier to read.

Reply:
@auduchinok, ok.